mdtraj/geometry/hbond.py

```diff
@@ -38,8 +37,7 @@
 # Functions
 ##############################################################################
 
-
-def wernet_nilsson(traj, exclude_water=True, periodic=True):
+def wernet_nilsson(traj, exclude_water=True, sidechain_only=False, periodic=True):
```

Comment:
Can you add `sidechain_only` to the end of this list. This makes sure it maintains API compatibility when using positional arguments